src/node_task_queue.cc

```diff
@@ -111,8 +112,17 @@ void PromiseRejectCallback(PromiseRejectMessage message) {
   }
 
   Local<Value> args[] = { type, promise, value };
+
+  // V8 does not expect this callback to have a scheduled exceptions once it
+  // returns, so we print them out in a best effort to do something about it
+  // without failing silently and without crashing the process.
```

Comment:
Maybe add a sentence about being near the stack size limit being a possible cause for an exception here?